locales/messages.json

```diff
@@ -1170,6 +1170,7 @@
   "_PortBugInvalidCrtLinkage.comment": "{expected} is one of LinkageDynamicDebug/LinkageDynamicRelease/LinkageStaticDebug/LinkageStaticRelease. Immediately after this message is a file by file list with what linkages they contain. 'CRT' is an acronym meaning C Runtime. See also: https://learn.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-170. This is complicated because a binary can link with more than one CRT.\nExample fully formatted message:\nThe following binaries should use the Dynamic Debug (/MDd) CRT.\n    C:\\some\\path\\to\\sane\\lib links with: Dynamic Release (/MD)\n    C:\\some\\path\\to\\lib links with:\n        Static Debug (/MTd)\n        Dynamic Release (/MD)\n    C:\\some\\different\\path\\to\\a\\dll links with:\n        Static Debug (/MTd)\n        Dynamic Debug (/MDd)\n",
   "PortBugInvalidCrtLinkageEntry": "{path} links with:",
   "_PortBugInvalidCrtLinkageEntry.comment": "See explanation in PortBugInvalidCrtLinkage An example of {path} is /foo/bar.",
+  "PortBugKernel32FromXbox": "The selected triplet targets Xbox, but the following DLLs link with kernel32. These DLLs cannot be loaded on Xbox, where kernel32 is not present. This is typically caused by linking with kerne32.lib rather than a suitable umbrella library, such as onecore_apiset.lib or xgameplatform.lib.",
```

Comment:
```suggestion
  "PortBugKernel32FromXbox": "The selected triplet targets Xbox, but the following DLLs link with kernel32. These DLLs cannot be loaded on Xbox, where kernel32 is not present. This is typically caused by linking with kernel32.lib rather than a suitable umbrella library, such as onecore_apiset.lib or xgameplatform.lib.",
```

Reply:
Thanks!